WORKSPACE

```diff
@@ -20,10 +32,6 @@ load("@io_bazel_rules_docker//docker:docker.bzl", "docker_repositories")
 
 docker_repositories()
 
-load(":adapter_author_deps.bzl", "mixer_adapter_repositories")
-
-mixer_adapter_repositories()
-
```

Comment:
I think this should remain.

Reply:
We cannot redefine dependencies after load. These are already defined in API dependencies.

Reply:
maybe @douglas-reid should know the answer -- I think it was added for custom (external) adapters, but still I don't know what it should look like with this PR.